From dae4ed28d87c4752aad768dbc4fbacae3a48c7ec Mon Sep 17 00:00:00 2001
From: gargi roy <Gargi.Roy@ust.com>
Date: Thu, 19 Jun 2025 09:51:28 +0530
Subject: [PATCH] Updated g toG

---
 helper.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/helper.py b/helper.py
index e408dd0..39ef05d 100644
--- a/helper.py
+++ b/helper.py
@@ -2,4 +2,4 @@ def greet():
   print('Hello! This is Gargi')
 
 def goodbye():
-  print('goodbye!')
+  print('Goodbye!')
-- 
2.50.0.windows.1

